R/PPI/results_analysis/models_submissions.R

```diff
@@ -0,0 +1,42 @@
+
+correspondance_entries <- list(
+  
+  'september' = list(
+    "entry_1" = "Naive",
+    "entry_2" = "Naive_2",
+    "entry_3" = "ARIMA"
+  ),
+  
+  'october' = list(
+    "entry_1" = "Naive",
+    "entry_2" = "ARIMA",
+    "entry_3" = "XGBoost",
+    "entry_4" = "DFM",
+    "entry_5" = "ETS"
+  ),
+  
+  'november' = list(
+    "entry_1" = "Naive",
+    "entry_2" = "REG-ARIMA",
+    "entry_3" = "XGBoost",
+    "entry_4" = "DFM",
+    "entry_5" = "ETS"
+  ),
+  
+  'december' = list(
+    "entry_1" = "REG-ARIMA",
+    "entry_2" = "XGBoost",
+    "entry_3" = "XGBoost_diff",
+    "entry_4" = "DFM",
+    "entry_5" = "ETS"
+  ),
+  
```

Comment:
Je pense ca serait mieux d'avoir ces infos dans un fichier config type yaml in fine